2.7.0/todo/result_distributed.md

```diff
@@ -30,7 +30,10 @@ The categories below are as follows:
 ### deprecation
 ### new features
 - Context parallel
-  - [place holder for CP description] ([#132820](https://github.com/pytorch/pytorch/pull/132820), [#142093](https://github.com/pytorch/pytorch/pull/142093), [#148377](https://github.com/pytorch/pytorch/pull/148377))
+  - Implemented Ring Attention ([#131351](https://github.com/pytorch/pytorch/pull/131351))
+  - Implemented AllGather-based Context Parallel where the all-gather happens after the local SDPA ([#132820](https://github.com/pytorch/pytorch/pull/132820)) and added user API to select the desired approach ([#142093](https://github.com/pytorch/pytorch/pull/142093))
+  - 3 variants of SDPA kernels are supported: `SDPBackend.FLASH_ATTENTION`, `SDPBackend.EFFICIENT_ATTENTION`, and `SDPBackend.CUDNN_ATTENTION` ([#148537](https://github.com/pytorch/pytorch/pull/148537))
+  - Compatible with other parallelisms and torch.compile
```

Comment:
usually the way we do this for new release is that we have one paragraph and you combine all PRs together or you just aggregate them somehow not using bullet points.